media/js/products/vpn/ppa-experiment.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+if (/\sFirefox/.test(navigator.userAgent)) {
+    const links = document.getElementsByClassName('js-fxa-product-button');
+    Array.from(links).forEach((link) => {
+        'use strict';
+
+        link.addEventListener('click', () => {
+            navigator.privateAttribution.measureConversion({
```

Comment:
Will this throw an error in older Firefox versions? It might be worth wrapping in a try/catch if so?